scripts/makepeds

```diff
@@ -222,7 +222,7 @@ else
     elif [[ $FFB == 2 ]]; then
         echo calling on SDF system: makepeds_psana $@
 	#we need to change paths here as directories are completely independent
-	DIR=`echo $DIR | sed 's/\/cds\/group\/pcds/sdf\/group\/lcls\/tools/g' | sed 's/\/reg\/g\/pcds/\/sdf\/group\/lcls\/ds\/tools/g'`
+	DIR=`echo $DIR | sed 's/\/cds\/group\/pcds/\/sdf\/group\/lcls\/tools/g' | sed 's/\/reg\/g\/pcds/\/sdf\/group\/lcls\/ds\/tools/g'`
```

Comment:
Does the 'cds' replace not also need the 'ds' at lcls/ds/tools?

Reply:
Also, lines 205-208 can go as they are doing what lines 2245 et al now do better. I could not figure out how to comment there.

Reply:
That's right!
ANd I will check the lines you mention. You mean line 245 though, right?